### ISSUE TITLE
osx-sign.gatekeeper-assess boolean option passed as string

### ISSUE THREAD

**Issue by acnghm** [electron/electron-packager#1179]
### Preflight Checklist

* [x] I have read the [contribution documentation](https://github.com/electron/electron-packager/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [code of conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project follows, as appropriate.
* [x] I have searched the issue tracker for a bug that matches the one I want to file, without success.

### Issue Details

* **Electron Packager Version:**
  * 15.1.0
* **Electron Version:**
  * v5.0.0
* **Operating System:**
  * macOS 10.15.6
* **Last Known Working Electron Packager version:**:
  * unknown

### Expected Behavior
Using the cli option `--osx-sign.gatekeeper-assess=false` should pass `gatekeeper-assess` option as boolean false to electron-osx-sign.

### Actual Behavior
Using the cli option `--osx-sign.gatekeeper-assess=false` seems to pass as option to electron-osx-sign as the string "false". This seems to cause the option to be interpreted as true (or perhaps defaults to true).

### To Reproduce
Running the following command (or similar) with the debug options should show the following line:

DEBUG=electron-osx-sign,electron-packager,electron-notarize* electron-packager ./ \
--out=./build \
--platform=darwin \
--overwrite \
--osx-sign.hardenedRuntime \
--osx-sign.gatekeeper-assess=false

```
  electron-packager Running electron-osx-sign with the options {"hardenedRuntime":true,"gatekeeper-assess":"false","platform":"darwin","app":"/var/folders/w6/p34bq97914b543d6j2j4ml15htrcjf/T/electron-packager/darwin-x64/Studio Uploader-darwin-x64/Studio Uploader.app","version":"5.0.0"} +7ms
```

### Additional Information

I hacked a fix locally so I could pass this option correctly by adding `osx-sign.gatekeeper-assess` as a boolean option to the array in cli.js [here](https://github.com/electron/electron-packager/blob/master/src/cli.js#L21-L29).

After doing this I see the following line when running the above command
```
  electron-packager Running electron-osx-sign with the options {"hardenedRuntime":true,"gatekeeper-assess":false,"platform":"darwin","app":"/var/folders/w6/p34bq97914b543d6j2j4ml15htrcjf/T/electron-packager/darwin-x64/Studio Uploader-darwin-x64/Studio Uploader.app","version":"5.0.0"} +7ms
```
Basically the same as before but `"gatekeeper-assess":false` instead of `"gatekeeper-assess":"false"`.

I have filed as a bug but I am unsure if this is a supported arg when used via electron-packager. I couldn't see it as a [supported and documented arg for osx-sign](https://github.com/electron/electron-packager/blob/master/usage.txt#L83-L90). Or perhaps there is a different way to pass this arg in?

**Comment by malept** [comment]
You want `--no-osx-sign.gatekeeper-assess` I believe.
